webpack.config.js

```diff
@@ -151,5 +151,11 @@ module.exports = {
 		] ),
 		new DependencyExtractionWebpackPlugin( { injectPolyfill: true } ),
 	],
+	resolve: {
+		alias: {
+			// Added for compatibility with IE.
+			'react-spring$': 'react-spring/web.cjs',
```

Comment:
This approach don't solve the issue in the root packages though. And we'd have to duplicate the fix in Core's webpack config file. I wonder if we should be explicit in the imports instead?

Reply:
Then we'll have to specify `react-spring/web.cjs` in every single import, right? Or am I missing something?

Reply:
Yes probably, we could try to write a custom eslint rule to prevent root imports.

Reply:
re eslint: it should be possible to use `no-restricted-syntax` rule for that.

Reply:
Done!